### PR TITLE
Added a new rule to scan for bucket with allUsers in IAM policy

### PR DESCRIPTION
**Pull request by red2k18**
Added a new rule to scan for bucket with allUsers in IAM policy

### CODE REVIEW

File: rules/iam_rules.yaml [hunk 1]
```diff
@@ -25,8 +25,22 @@ rules:
     bindings:
       - role: roles/resourcemanager.organizationAdmin
         members:
-          - user:*@{DOMAIN}
-          - group:*@{DOMAIN}
+          - user:*@henrychang.mygbiz.com
```
Comment (blueandgold) [inline rules/iam_rules.yaml]: Please add the placeholders back.

File: rules/iam_rules.yaml [hunk 2]
```diff
@@ -25,8 +25,22 @@ rules:
...
+          - group:*@henrychang.mygbiz.com
+
+  - name: Prevent allUsers access to buckets via IAM
```
Comment (blueandgold) [inline rules/iam_rules.yaml]: I would describe this name as:

```
Prevent public users from having access to buckets via IAM
```

Comment (red2k18) [reply rules/iam_rules.yaml]: Done

File: rules/iam_rules.yaml [hunk 3]
```diff
@@ -28,6 +28,20 @@ rules:
...
+        applies_to: self
+        resource_ids:
+          - '*'
+    inherit_from_parents: true
+    bindings:
+      - role: '*'
+        members:
+          - allUsers
+
+
```
Comment (blueandgold) [inline rules/iam_rules.yaml]: One newline here only.

**Pull request merged**